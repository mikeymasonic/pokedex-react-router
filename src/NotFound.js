import React, { Component } from 'react'

export default class NotFound extends Component {
    render() {
        return (
            <div>
                <h2>404 Not found! o nooo</h2>
            </div>
        )
    }
}
