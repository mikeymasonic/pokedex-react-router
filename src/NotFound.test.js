import React from 'react';
import NotFound from './NotFound';
import renderer from "react-test-renderer";
test('renders NotFound.js correctly', () => {
  const tree = renderer
  .create(<NotFound />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});