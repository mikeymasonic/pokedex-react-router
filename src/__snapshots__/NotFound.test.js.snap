// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NotFound.js correctly 1`] = `
<div>
  <h2>
    404 Not found! o nooo
  </h2>
</div>
`;
